.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
```

Comment:
I think we should update the doc to mention lowercased `bug_report.md` and `feature_request.md`, because:
- That's what we have in our dotnet template repo
- Microsoft's and even Github's repo use lowercase too

![image](https://user-images.githubusercontent.com/14242083/210824538-fe6f4fd2-e827-41af-898f-ba2cb2a6c2d6.png)

![image](https://user-images.githubusercontent.com/14242083/210824583-946ffdfa-6633-4810-b890-cdc7ad6f01a8.png)

Reply:
Sounds good then!